tests/pytests/unit/states/test_win_path.py

```diff
@@ -0,0 +1,496 @@
+"""
+Tests for win_path states
+"""
+
+import copy
+
+import pytest
+import salt.states.win_path as win_path
+from tests.support.mock import MagicMock, Mock, patch
+
+NAME = "salt"
```

Comment:
Would prefer not to have a module level variable here (per docs: https://docs.saltproject.io/en/master/topics/development/tests/index.html#module-global-level-variables) unless we have to.